doc/TOC.md

```diff
@@ -0,0 +1,34 @@
+The DataStax Node.js Driver for Apache Cassandra is a modern [feature-rich](#features) and highly tunable Node.js
```

Comment:
It doesn't look like there is a #features part of this page, did you mean to link to submodules?